src/settings.ts

```diff
@@ -1,6 +1,7 @@
 const defaultSettings: Settings = {
-    background: "#FAFAFA",
+    background: "#fafafa",
     foreground: "#424242",
+    colorScheme: "default",
```

Comment:
```suggestion
    colorScheme: "custom",
```
This should be `custom` to handle already existing custom colors.